src/Coordination/KeeperStorage.cpp

```diff
@@ -375,26 +375,26 @@ void KeeperStorage::UncommittedState::applyDelta(const Delta & delta)
         delta.operation);
 }
 
-void KeeperStorage::UncommittedState::addDelta(Delta new_delta)
+void KeeperStorage::UncommittedState::addDelta(const Delta & new_delta)
 {
-    const auto & added_delta = deltas.emplace_back(std::move(new_delta));
```

Comment:
It is tricky that `addDelta` doesn't properly add deltas. Probably it's good to have a comment about it